openapi-cli/src/test/resources/expected_gen/generatedRB.bal

```diff
@@ -0,0 +1,13 @@
+service / on ep0 {
+
+#
+# + caller - Caller client object represents the endpoint
+# + req    - Req represents the message which came along over the network
+
+# + payload - Request body payload.
+# + return - Error value if an error occurred, and return nil otherwise
+    resource function post requestBody(http:Caller caller, http:Request req
+    , @http:Payload {} User  payload) returns error? {}
```

Comment:
Where is the user record ?

Reply:
The user record is generated with the separate schema file.